crates/settings/src/settings_file.rs

```diff
@@ -15,12 +15,14 @@ pub fn test_settings() -> String {
     .unwrap();
     util::merge_non_null_json_value_into(
         serde_json::json!({
-            "ui_font_family": "Courier",
+            "ui_font_family": ["Courier"],
```

Comment:
```suggestion
            "ui_font_family": "Courier",
```

I guess this fails the tests.

Reply:
ugh, I can't believe I missed that. Thanks!